src/main/scala/mesosphere/marathon/core/task/termination/impl/KillServiceActor.scala

```diff
@@ -93,23 +101,27 @@ private[impl] class KillServiceActor(
 
   def killUnknownTaskById(taskId: Task.Id): Unit = {
```

Comment:
why can't this go through `killInstances`?

Reply:
We don't have an instance for unknown tasks, that's all.

Reply:
what he said ^.^